tests/test_format.py

```diff
@@ -87,3 +21,5 @@ def test_conformance(paths=paths):
     style = pycodestyle.StyleGuide(quiet=False, max_line_length=120)
     result = style.check_files(paths)
     assert result.total_errors == 0
+
```

Comment:
you need to add one more blank line here

Reply:
added